src/MPILandscape.jl

```diff
@@ -33,6 +34,10 @@ mutable struct MPIGridLandscape{RA <: Base.ReshapedArray, NT <: NamedTuple}
       reshape(view(cols_vector, (sppindices[i] + 1) : sppindices[i + 1]),
               Int64(sppcounts[i]), Int64(sccounts[rank + 1]))
     end
+    reshaped_cols_vector = vcat(map(1:length(sccounts)) do i
+      reshape(view(cols_vector, (sppindices[i] + 1) : sppindices[i + 1]),
+              Int64(sppcounts[i]), Int64(sccounts[rank + 1]))
+    end ...)[1:end]
```

Comment:
So you create this `reshaped_cols_vector` here, but you don't seem to use it... is it supposed to be used here:

https://github.com/boydorr/EcoSISTEM.jl/blob/09e21e946e03ce9e9ad1cc862319df94723a7fe3/src/MPIEcosystem.jl#L110

If so (it looks a bit different?) you haven't edited the code there.

Reply:
Oh dear, no wonder it worked! Sorry, I'm having a bit of trouble with this one - I can't get it to sync up properly